sample/src/debug/java/com/polidea/cockpit/sample/MainActivity.kt

```diff
@@ -1,13 +1,24 @@
 package com.polidea.cockpit.sample
 
 import android.view.View
+import android.widget.Toast
 import com.polidea.cockpit.cockpit.Cockpit
+import com.polidea.cockpit.event.PropertyChangeListener
 
 class MainActivity : VariantIndependentBaseMainActivity() {
 
     override fun initViews() {
         findViewById<View>(R.id.edit_values_button).setOnClickListener {
             Cockpit.showCockpit(this)
         }
+
+        Cockpit.addOnFontSizeChangeListener(
```

Comment:
Maybe we could use our property change listener mechanism in sample app and remove `VariantIndependentBaseMainActivity.displayMainActivity()` called from `Activity.onResume` and add there `fontSizeChangeListener`, `textColorChangeListener` etc, hmm?